rkt/rkt.mk

```diff
@@ -32,3 +32,11 @@ include makelib/build_go_bin.mk
 $(call undefine-namespaces,LOCAL)
 # RKT_STAMP deliberately not cleared
 # RKT_BINARY deliberately not cleared
+
+manpages:
```

Comment:
> I would make it a part of make all.

It would make the compilation quite slow if we run `make manpages/bash-completion` at each `make` though... On my laptop:

- time make manpages: 25 seconds
- time make bash-completion: 24 seconds
- total: 49 seconds

> I would prefer the files to be generated inside $(BUILDDIR)
> That would require passing that directory to the generator, so it would know where to put the files.
> But that complicates things (make clean, deps, forwarding variables, blah blah blah), so don't bother with it for now.

Ok, I'll skip that for now.

> Pretty printers could be used there.
> No big deal, since the output is rather short.

How to do that?

> These additions should be put above the $(call undefine-namespaces,LOCAL) line.

Will do in the next update.